modules/vaos/lib/vaos/exceptions/validation_errors.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'common/exceptions/base_error'
+require 'common/exceptions/serializable_error'
+
+module VAOS
+  module Exceptions
+    class ValidationErrors < Common::Exceptions::BaseError
```

Comment:
The base error class selects the i18n key based on the class name `ValidationErrors` => `common.exceptions.validation_errors`